sdk/keyvault/Azure.Security.KeyVault.Secrets/Readme.md

```diff
@@ -0,0 +1,255 @@
+
+ * Create a service principal and configure its access to Azure resources:
+    ```PowerShell
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+    Output:
+    ```json
+    {
+        "appId": "generated-app-ID",
+        "displayName": "dummy-app-name",
+        "name": "http://dummy-app-name",
+        "password": "random-password",
+        "tenant": "tenant-ID"
+    }
+    ```
+* Use the above returned credentials information to set **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) environment variables. The following example shows a way to do this in Powershell:
+```cmd
+$Env:AZURE_CLIENT_ID="generated-app-ID"
+$Env:AZURE_CLIENT_SECRET="random-password"
+$Env:AZURE_TENANT_ID="tenant-ID"
+```
+
+* Grant the above mentioned application authorization to perform secret operations on the key vault:
+    ```PowerShell
+    az keyvault set-policy --name <your-key-vault-name> --spn $AZURE_CLIENT_ID --secret-permissions backup delete get list set
+    ```
+    > --secret-permissions:
+    > Accepted values: backup, delete, get, list, purge, recover, restore, set
+
+* Use the above mentioned Key Vault name to retreive details of your Vault which also contains your Key Vault URL:
+    ```PowerShell
+    az keyvault show --name <your-key-vault-name> 
+    ```
+
+#### Create Secret client
+Once you've populated the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and **AZURE_TENANT_ID** environment variables and replaced **your-vault-url** with the above returned URI, you can create the [SecretClient][secret_client_class]:
+
+```c#
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+
+# Create a new secret client using the default credential from Azure.Identity
+var client = new SecretClient(vaultUri: <your-vault-url>, credential: new SystemCredential())
+```
+
+## Key concepts
+### Secret
+A secret is the fundamental resource within Azure KeyVault. From a developer's perspective, Key Vault APIs accept and return secret values as strings. In addition to the secret data, the following attributes may be specified:
+* Expires: Identifies the expiration time on or after which the secret data should not be retrieved.
+* NotBefore: Identifies the time after which the secret will be active.
+* Enabled: Specifies whether the secret data can be retrieved.
+* Created: Indicates when this version of the secret was created.
+* Updated: Indicates when this version of the secret was updated.
+
+### Secret Client:
+The Secret client performs the interactions with the Azure Key Vault service for getting, setting, updating, deleting, and listing secrets and its versions. An asynchronous and synchronous SecretClient client exists in the SDK allowing for selection of a client based on an application's use case. Once you've initialized a SecretClient, you can interact with the primary resource types in Key Vault.
```

Comment:
This seems like a repeat of what was said at the very beginning of the readme.

Reply:
Fair. I deleted the repetition